### ISSUE TITLE
Adds double newline when `endOfLine: crlf` is configured in prettier

### ISSUE THREAD

**Issue by n2k3** [ony3000/prettier-plugin-merge#31]
## Dependency information
WebStorm (with "run prettier on save" enabled) on Windows

## Steps to reproduce
I have this prettier config in `package.json`:
```json
  "prettier": {
    "printWidth": 120,
    "endOfLine": "crlf",
    "plugins": ["prettier-plugin-merge"]
  }
```
<sup>Note: usually `prettier-plugin-tailwindcss` and `prettier-plugin-classnames` are in the included at the start of the array, but I omitted them as the issue is causes by the `prettier-plugin-merge` plugin.</sup>

## The current behavior
![image](https://github.com/user-attachments/assets/b7c8b7ac-fceb-42c9-b33b-59e4ab977a2e)

## The expected behavior
![image](https://github.com/user-attachments/assets/7b96e527-f8af-4585-9166-1288445efe14)

**Comment by ony3000** [comment]
Hello! Thanks for reporting the issue!

Unfortunately, I've only used Webstorm a little bit, so it's difficult to reproduce this issue with Webstorm.

Can you confirm that this is reproduced in VSCode with the same Prettier settings and code? If setting up VSCode is difficult, can you provide me with the minimum repository to reproduce the issue?

**Comment by osh6006** [comment]
I got the same bug. Below is my .pretierrc.json file.

{
  "endOfLine": "auto",
  "printWidth": 80,
  "tabWidth": 2,
  "useTabs": false,
  "semi": true,
  "singleQuote": true,
  "singleAttributePerLine": true,
  "bracketSpacing": true,
  "arrowParens": "always",
  "trailingComma": "es5",
  "plugins": [
    "prettier-plugin-tailwindcss",
    "prettier-plugin-classnames",
    "@trivago/prettier-plugin-sort-imports",
    "prettier-plugin-merge"
  ],
  "importOrder": [
    "^react",
    "^next",
    "^lucide-react",
    "<THIRD_PARTY_MODULES>",

    "^~/service/(.*)$",
    "^~/hooks/(.*)$",
    "^~/provider/(.*)$",
    "^~/components/(.*)$",
    "^~/utils/(.*)$",

    "^[./]"
  ],
  "importOrderSeparation": true,
  "importOrderSortSpecifiers": true
}

**Comment by ony3000** [comment]
@n2k3 @osh6006 Fixed in `v0.7.1`, check it out!
